app/scripts/templates/settings.mustache

```diff
@@ -1,37 +1,31 @@
-<div id="main-content" class="card">
-  <header>
-    <h1 id="fxa-settings-header">{{#t}}Manage account{{/t}}</h1>
+<div id="fxa-settings-header-wrapper">
+  <header class="fxa-settings-header">
+    <h1 id="fxa-manage-account">{{#t}}Manage account{{/t}}</h1>
+    {{#showSignOut}}
+    <button id="signout" class="settings-button secondary">Sign Out</button>
```

Comment:
`Sign Out` or `Sign out`? (this page says "Manage account" not "Manage Account", not sure if the "Out" is weird).
/cc @ryanfeeley @johngruen